app/assets/javascripts/components/student/student_edit.es6.jsx

```diff
@@ -33,6 +33,7 @@ class StudentEdit extends Component {
         top: '0px',
         right: '8px',
         lineHeight: '40px',
+        cursor: 'pointer',
```

Comment:
See comment below, if it's a `Clickable` you won't need to set `cursor: 'pointer'`.